projects/angular/src/accordion/accordion-panel.ts

```diff
@@ -52,12 +61,23 @@ export class ClrAccordionPanel implements OnInit, OnChanges {
   private _panelIndex: number;
 
   constructor(
+    @Optional()
+    @SkipSelf()
+    protected parent: ClrAccordionPanel,
     public commonStrings: ClrCommonStringsService,
     private accordionService: AccordionService,
     private ifExpandService: IfExpandService,
     private cdr: ChangeDetectorRef
   ) {}
 
+  get headingLevel() {
+    if (this.explicitHeadingLevel) {
+      return this.explicitHeadingLevel;
+    }
+
+    return this.parent ? 4 : 3;
+  }
```

Comment:
nit: It feels odd to me to put this before the `id` property.

Reply:
Corrwcted in commit 2cb2400